service/src/test/java/greencity/service/UserServiceImplTest.java

```diff
@@ -1187,9 +1187,11 @@ void searchBy() {
     }
 
 
-    /*@Test
+    /*
```

Comment:
Bad

Reply:
not mine edited to pass through checkstyle